app/controllers/course/statistics/answers_controller.rb

```diff
@@ -33,7 +33,7 @@ def all_answers
     @question_index = question_index(question_id)
     @all_answers = Course::Assessment::Answer.
                    unscope(:order).
-                   order(:created_at).
+                   order(:submitted_at).
```

Comment:
Since we also want to display submissions in attempting state, what happens to answers with `nil` for `submitted_at`?

Reply:
hm I dont think there is a way to reconcile that. But do you think its ok for us to order by `created_at` and also send the `created_at` time but present it to the user as `submitted_at` (which is what was done previously

Reply:
It's ok, since user has not run the code, it will not be counted as an attempt, so it will reflect as 0 on attempts page, (or whatever number depending on how many past attempts they already had existing).

Reply:
Ok changed to order by created_at